extensions/positron-python/.github/workflows/build.yml

```diff
@@ -254,6 +254,13 @@ jobs:
       - name: Install test requirements
         run: python -m pip install --upgrade -r build/test-requirements.txt
 
+      - name: Rust Tool Chain setup
```

Comment:
I don't think this should affect our build, but noting in case we should comment it out until we use the `python-environment-tools` Python locator

Reply:
Yeah I _think_ we don't use the workflows in this directory, so this is should be okay as-is!